imports/startup/client/useraccounts-configuration.js

```diff
@@ -1,15 +1,16 @@
 import { AccountsTemplates } from 'meteor/useraccounts:core';
+import { TAPi18n } from 'meteor/tap:i18n'
 
 AccountsTemplates.configure({
   showForgotPasswordLink: true,
   texts: {
     errors: {
-      loginForbidden: 'Incorrect username or password',
-      pwdMismatch: 'Passwords don\'t match',
+      loginForbidden: TAPi18n.__('Incorrect username or password'),
```

Comment:
This doesn't seem right - I'm sure there is an article somewhere on i18n with useraccounts...

https://github.com/meteor-useraccounts/core/blob/master/Guide.md#i18n